receiver/auto.go

```diff
@@ -63,6 +63,11 @@ func init() {
 		Alloc: func() interface{} { return &MQTT{} },
 		Help:  "Listen for Skogul-formatted JSON on a MQTT endpoint.",
 	})
+	Auto.Add(skogul.Module{
+		Name:  "stats",
+		Alloc: func() interface{} { return &Stats{} },
+		Help:  "Periodically ship Skogul stats.",
```

Comment:
This definitely needs to be a bit more verbose. 

Suggestion: "Gather internal Skogul telemetry and ship it. What telemetry is gathered depends on the modules used. Some modules send no stats at all, while others might log things such as number of parse errors or number of sent messages."